cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -1400,15 +1400,15 @@ if (ARROW_WITH_GRPC)
 
     set(GRPC_CMAKE_ARGS
       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
-      -DCMAKE_PREFIX_PATH="${GRPC_PREFIX_PATH_ALT_SEP}"
-      "-DgRPC_CARES_PROVIDER=package"
-      "-DgRPC_GFLAGS_PROVIDER=package"
-      "-DgRPC_PROTOBUF_PROVIDER=package"
-      "-DgRPC_SSL_PROVIDER=package"
-      "-DgRPC_ZLIB_PROVIDER=package"
-      "-DCMAKE_CXX_FLAGS=${EP_CXX_FLAGS}"
-      "-DCMAKE_C_FLAGS=${EP_C_FLAGS}"
-      "-DCMAKE_INSTALL_PREFIX=${GRPC_PREFIX}"
+      -DCMAKE_PREFIX_PATH='${GRPC_PREFIX_PATH_ALT_SEP}'
+      '-DgRPC_CARES_PROVIDER=package'
+      '-DgRPC_GFLAGS_PROVIDER=package'
+      '-DgRPC_PROTOBUF_PROVIDER=package'
+      '-DgRPC_SSL_PROVIDER=package'
+      '-DgRPC_ZLIB_PROVIDER=package'
+      '-DCMAKE_CXX_FLAGS=${EP_CXX_FLAGS}'
+      '-DCMAKE_C_FLAGS=${EP_C_FLAGS}'
+      '-DCMAKE_INSTALL_PREFIX=${GRPC_PREFIX}'
```

Comment:
Hm I wonder if this works on Windows.

Reply:
Ah that might not. On the other hand, on MacOS the double quotes got included in gRPC's path (so it tried to find `/path/to/grpc"`).

Reply:
Removing them seems OK. I'll test on Windows soon